### DISCUSSION TITLE
Why Scratch Addons Doesn't Work

### DISCUSSION THREAD

**Discussion by Ma4ve** [ScratchAddons/ScratchAddons#5364] [Help]
when i open scratch addons, it shows totally empty, why this happends?

**Comment by Endermaster3690** [comment]
related #5289

**Comment by Endermaster3690** [comment]
please rename to something better too

**Comment by BroJac5246** [comment]
Can you provide more detail? Perhaps some screenshots? Did you just download the extension, or did this happen out of the blue? We can't help you if we don't know what is going on.

**Comment by iqnite** [comment]
Reinstalling is always a good option. Remember to back up your settings first!

**Comment by DNin01** [comment]
If this is anything like the other reports of Scratch Addons not working that I've seen, it may be fixable with a browser or computer restart.
